### ISSUE TITLE
Menu bug with chests

### ISSUE THREAD

**Issue by neonvortex** [ArnoAnsems/CatacombGL#27]
In Catacomb Abyss 1.13 shareware, if the menu is opened after the player opens a chest, the player can move instead of being frozen in place while the items enter their inventory.

**Comment by neonvortex** [comment]
I did some testing and figured out that I was wrong about the effects of the chest.
It actually pauses the game, so I think the issue is that the menu un-pauses the game even while the chest's effect is active.

**Comment by ArnoAnsems** [comment]
Thanks for reporting this! I just reproduced the problem. When I pressed ESC while picking up a chest, I was still retrieving items from the chest AND got attacked by a zombie, all while the menu is visible. So that's indeed a bug.
I checked the behavior on the original Catacomb Abyss with DOSBox. There it just ignores the ESC button while picking up a chest, That's not a solution I would prefer. 
I will fix it by really pausing the game when the menu is activated during chest pickup. When exiting the menu, it should resume with retrieving items from the chest.
